mlir/lib/Dialect/GPU/IR/GPUDialect.cpp

```diff
@@ -2564,7 +2564,7 @@ CompilationTarget TargetOptions::getDefaultCompilationTarget() {
 }
 
 std::pair<llvm::BumpPtrAllocator, SmallVector<const char *>>
-TargetOptions::tokenizeCmdOptions() const {
+TargetOptions::tokenizeCmdOptions(const std::string &cmdOptions) {
   std::pair<llvm::BumpPtrAllocator, SmallVector<const char *>> options;
   llvm::StringSaver stringSaver(options.first);
   StringRef opts = cmdOptions;
```

Comment:
What happened to the original `TargetOptions.cmdOptions`? Is this now ignored? Should it be removed?

Reply:
Oh this is an overload now.